source/MicroBitAudio.cpp

```diff
@@ -55,12 +55,17 @@ MicroBitAudio::MicroBitAudio(NRF52Pin &pin, NRF52Pin &speaker, NRF52ADC &adc, NR
     if (MicroBitAudio::instance == NULL)
         MicroBitAudio::instance = this;
 
+    // Request a periodic callback
+    status |= DEVICE_COMPONENT_STATUS_SYSTEM_TICK;
+
     synth.allowEmptyBuffers(true);
 
     mic = adc.getChannel(microphone, false);
-    adc.setSamplePeriod( 1e6 / CONFIG_MIXER_DEFAULT_CHANNEL_SAMPLERATE );
+    mic->setSampleRate(11000);
```

Comment:
Should the default rate be a config flag? These flags have been useful when debugging issues in MakeCode as they can be set via pxt.json file.

Reply:
Thanks Joe!